packages/headless/src/controllers/core/generated-answer/headless-core-interactive-citation.ts

```diff
@@ -0,0 +1,64 @@
+import {GeneratedAnswerCitation} from '../../../api/generated-answer/generated-answer-event-payload';
+import {CustomAction} from '../../../features/analytics/analytics-utils';
+import {CoreEngine} from '../../../product-listing.index';
+import {
+  buildInteractiveResultCore,
+  InteractiveResultCore,
+  InteractiveResultCoreOptions,
+  InteractiveResultCoreProps,
+} from '../../core/interactive-result/headless-core-interactive-result';
+
+export interface InteractiveCitationOptions
+  extends InteractiveResultCoreOptions {
+  /**
+   * The generated answer citation.
+   */
+  citation: GeneratedAnswerCitation;
+}
+
+export interface InteractiveCitationProps extends InteractiveResultCoreProps {
```

Comment:
```suggestion
export interface InteractiveCitationCoreProps extends InteractiveResultCoreProps {
```

?

Reply:
Similar as above, since those types are ultimately re-exported from index globally, the external name must not change or that would be considered a breaking change.

It's possible to rename here, but it means we have to re-create the old type name (which can then be just a copy of core options) before re-exported it globally.

Since there's no benefit, I would suggest just leaving it named `InteractiveCitationProps` here

Reply:
ok makes sense! no need for change then!